i18n/locales/source/course_content/2020/pre-express-2020.json

```diff
@@ -700,11 +601,110 @@
       "courseA_artist_loops_challenge2_c": "All of your blocks will need to be inside of a loop to finish this puzzle."
     }
   },
-  "https://studio.code.org/s/pre-express-2020/stage/9/extras?level_name=courseA_artist_loops_challenge1_2020": {
+  "https://studio.code.org/s/pre-express-2020/stage/8/extras?level_name=courseA_artist_loops_challenge1_2020": {
     "short_instructions": "Can you draw a house? Make it look just the way you like it!",
     "long_instructions": "You can use a loop to draw many lines in a row.\n\nCan you draw a house? Make it look just the way you like it!  Try looking for patterns that repeat!\n\nHere are some ideas:\n\n[][0]",
     "teacher_markdown": "This lesson extra is a “free play”. These levels have some characteristics you may want to explore before starting. \n\n-   These levels are **not checked for correctness** to allow for more open-ended creativity. Empower your students to determine for themselves when they have created something they like. There is no one right answer! \n-   Example solutions are available for this level which can be viewed by opening the teacher panel to the right."
   },
+  "https://studio.code.org/s/pre-express-2020/stage/9/puzzle/2": {
```

Comment:
Looks like the lessons got rearranged again

@uponthesun it looks like this may have been from [this merge commit](https://github.com/code-dot-org/code-dot-org/commit/25b5093fd8ad5b1201dbbefc38dc9bf6a1878436#diff-e237311b0734c77427d91ac8091aad34). Do you know if this is intentional?

Reply:
Not intentional. Sorry!

Reply:
Is there a plan to revert this change?